src/rat/data_processing/newdata.py

```diff
@@ -56,7 +56,9 @@ def _determine_precip_link_and_version(date):
     if version == "IMERG-FINAL":
         link = f"ftp://arthurhou.pps.eosdis.nasa.gov/gpmdata/{date.strftime('%Y')}/{date.strftime('%m')}/{date.strftime('%d')}/gis/3B-DAY-GIS.MS.MRG.3IMERG.{date.strftime('%Y%m%d')}-S000000-E235959.0000.V06A.tif"
     elif version == "IMERG-LATE":
-        if date >= datetime(2023, 11, 8): # Version was changed from V06D to V06E
+        if date >= datetime(2024, 6, 1): # Version was changed from V06E to V07B
```

Comment:
Should we also update the version for IMERG-FINAL or IMERG-EARLY? It seems IMERF-early at line 70 of new file was getting changed until version 6D.